Code/Tools/AssetProcessor/native/resourcecompiler/rcjob.cpp

```diff
@@ -928,6 +928,7 @@ namespace AssetProcessor
             const QString& productAbsolutePath = filePair.second;
 
             bool isCopyJob = !(sourceAbsolutePath.startsWith(tempFolder, Qt::CaseInsensitive));
+            isCopyJob |= response.m_keepTempFolder; // Copy instead of Move if the builder wants to keep the Temp Folder. 
```

Comment:
It may. But this feature is used for debugging, not for production so I wouldn't be concerned. Granted, per your suggestions below I'll add more comments to warn the developers that this flag is to help debugging issues and should not be used in production.